### PR TITLE
split out yoe stuff from local.conf

### PR DESCRIPTION
**Pull request by cbrake**
clean out most of local.conf for project use.

### CODE REVIEW

File: conf/yoe.conf [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+PREFERRED_PROVIDER_u-boot-fw-utils ??= "u-boot-fw-utils"
+
+
+# Don't generate the mirror tarball for SCM repos, the snapshot is enough
+BB_GENERATE_MIRROR_TARBALLS = "0"
+
+# By default disable interactive patch resolution (tasks will just fail instead):
+PATCHRESOLVE = "noop"
+
+DISTRO = "yoe"
```
Comment (kraj) [inline conf/yoe.conf]: Not needed.

File: conf/yoe.conf [hunk 2]
```diff
@@ -0,0 +1,57 @@
+# CONF_VERSION is increased each time build/conf/ changes incompatibly
+CONF_VERSION = "1"
+
+# Modify the following if you need to change the types of images generated
+# most of the time, this is set by the machine conf or in a custom image
+# recipe.
+IMAGE_FSTYPES_append = " tar.gz wic.xz"
```
Comment (kraj) [inline conf/yoe.conf]: Do we need tar.gz ? and May be just override it, drop appending to defaults.

File: conf/yoe.conf [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+#   PARALLEL_MAKE specifies how many concurrent compiler threads are spawned per bitbake process
+#   BB_NUMBER_THREADS specifies how many concurrent bitbake tasks will be run
+
+PREFERRED_PROVIDER_virtual/libgl ??= "mesa"
+PREFERRED_PROVIDER_virtual/psplash ??= "psplash"
+PREFERRED_PROVIDER_psplash-support ??= "psplash"
+PREFERRED_PROVIDER_jpeg ??= "libjpeg-turbo"
+PREFERRED_PROVIDER_jpeg-native ??= "libjpeg-turbo-native"
+PREFERRED_PROVIDER_grub-efi-native ??= "grub-efi-native"
+PREFERRED_PROVIDER_u-boot-fw-utils ??= "u-boot-fw-utils"
```
Comment (kraj) [inline conf/yoe.conf]: this should perhaps go into meta-yoe

File: conf/local.conf [hunk 4]
```diff
@@ -1,62 +1,10 @@
+# include yoe defaults, can be overriden in this file
+require yoe.conf
```
Comment (kraj) [inline conf/local.conf]: Perhaps move these settings to distro conf file in meta-yoe

File: conf/yoe.conf (hunk 2 above)
Comment (cbrake) [reply conf/yoe.conf]: tar.gz is useful for peeking what is in the rootfs -- there may be better ways to do this.

Comment (kraj) [reply conf/yoe.conf]: ah, I usually go into build tree for image recipe and look inside rootfs/ folder. but that does not fix the issue when you only have images around.

**Comment by cbrake** [comment]
updated this and matching pull request in meta-yoe.

**Pull request merged**